packages/editor/src/components/post-saved-state/index.js

```diff
@@ -154,7 +154,7 @@ export default function PostSavedState( {
 			shortcut={ displayShortcut.primary( 's' ) }
 			variant={ isLargeViewport ? 'tertiary' : undefined }
 			icon={ isLargeViewport ? undefined : cloudUpload }
-			label={ label }
+			label={ showIconLabels ? undefined : label }
```

Comment:
I'm not 100% sure that we need a label here. The button text should be enough from a11y perspective; this only serves as a visual enhancement for the tooltip.

Reply:
I think the aria-label was there because that button used to be just an icon on all breakpoints. As long as there's text in the button, the aria-label isn't needed.

Reply:
Do you recommend removing `label` or reverting changes from #38790?

Reply:
No, I think it's fine as it is; we still need to render the label when showing the icon button on mobile.